lib/blocks.php

```diff
@@ -466,5 +470,26 @@ function gutenberg_migrate_old_typography_shape( $metadata ) {
 	return $metadata;
 }
 
-
 add_filter( 'block_type_metadata', 'gutenberg_migrate_old_typography_shape' );
+
+/**
+ * Filters the content of a single block.
+ *
+ * @param array $args Array of arguments for registering a block type.
+ *
+ * @return array Returns the $metadata with any missing `style` and `editorStyle` added.
+ */
+function gutenberg_add_missing_styles_to_core_block_json( $args ) {
```

Comment:
Can we move this filter to `lib/compat/wordpress-5.8` folder:

https://github.com/WordPress/gutenberg/tree/trunk/lib/compat/wordpress-5.8

It will help with the maintenance of the Gutenberg plugin. Otherwise, it's a lot of detective work to find out when feature in the compatibility layer can get removed.